docs/guides/operations-guide/manager/apply.md

```diff
@@ -34,3 +38,56 @@ $ osism apply
 | iscsi                                                            | kolla          |
 ...
 ```
+
+## Apply a role
+
+```
+$ osism apply operator -l st01-mgmt-r01-u30
+2024-06-14 09:33:10 | INFO     | Task f94a2e6f-d199-421c-b7b7-743db4661305 (operator) was prepared for execution.
+2024-06-14 09:33:10 | INFO     | It takes a moment until task f94a2e6f-d199-421c-b7b7-743db4661305 (operator) has been started and output is visible here.
+
+PLAY [Make ssh pipelining working] *********************************************
+
+TASK [Do not require tty for all users] ****************************************
+ok: [st01-mgmt-r01-u30]
+
+...
+...
+
+TASK [osism.commons.operator : Set password of operator user] ******************
+ok: [st01-mgmt-r01-u30]
+
+TASK [osism.commons.operator : Set root password like operator password] *******
+ok: [st01-mgmt-r01-u30]
+
+PLAY RECAP *********************************************************************
+2024-06-14 09:34:14 | INFO     | Play has been completed. There may now be a delay until all logs have been written.
+2024-06-14 09:34:14 | INFO     | Please wait and do not abort execution.
+st01-mgmt-r01-u30          : ok=11   changed=0    unreachable=0    failed=0    skipped=5    rescued=0    ignored=0
+```
+
+## Use of custom plays
+
+Custom Plays can be used in all environments in the configuration repository.
+
+For example, this is a play to prepare all devices to be used for Ceph on a Ceph
+resource node. It is saved in the configuration repository in the file
+`environments/ceph/playbook-wipe-partitions.yml`. It is run with
+`osism apply -e ceph wipe-parititons`.
+
+```yaml title="environments/ceph/playbook-wipe-partitions.yml
+---
+- name: Wipe partitions
+  hosts: ceph-resource
+  gather_facts: false
+
+  tasks:
+    - name: Wipe partitions
+      become: true
+      ansible.builtin.shell: |
+        wipefs --all "{{ item }}"
+        dd if=/dev/zero of="{{ item }}" bs=1 count=4096
+      changed_when: false
+      with_items: "{{ devices }}"
```

Comment:
While touching this sync this sample with the play https://github.com/osism/testbed/blob/main/environments/custom/playbook-wipe-partitions.yml.